include/z64math.h

```diff
@@ -114,10 +128,17 @@ typedef struct {
 
 // Angle conversion macros
 #define DEG_TO_BINANG(degrees) (s16)((degrees) * (0x8000 / 180.0f))
-#define RADF_TO_BINANG(radf) (s16)((radf) * (0x8000 / M_PI))
-#define RADF_TO_DEGF(radf) ((radf) * (180.0f / M_PI))
-#define DEGF_TO_RADF(degf) ((degf) * (M_PI / 180.0f))
-#define BINANG_TO_RAD(binang) (((f32)binang / 0x8000) * M_PI)
+#define DEG_TO_BINANG_ALT(degrees) (s16)(((degrees) / 180.0f) * 0x8000)
+#define RAD_TO_BINANG(radians) (s16)((radians) * (0x8000 / M_PI))
+#define RAD_TO_BINANG_ALT(radians) (s16)((radians * 0x8000) / M_PI)
+
+#define RAD_TO_DEG(radf) ((radf) * (180.0f / M_PI))
+#define DEG_TO_RAD(degf) ((degf) * (M_PI / 180.0f))
+
+#define BINANG_TO_DEG(binang) ((f32)(binang) * (180.0f / 0x8000))
+#define BINANG_TO_RAD(binang) ((f32)(binang) * (M_PI / 0x8000))
+#define BINANG_TO_RAD_ALT(binang) (((f32)(binang) / (f32)0x8000) * M_PI)
```

Comment:
```suggestion
#define BINANG_TO_RAD_ALT(binang) (((f32)(binang) / 0x8000) * M_PI)
```
Afaik the cast shouldn't be needed